polling_stations/apps/data_collection/management/commands/import_dorset.py

```diff
@@ -1,142 +1,122 @@
 from django.contrib.gis.geos import Point
-from data_collection.management.commands import (
-    BaseXpressDCCsvInconsistentPostcodesImporter,
-)
+from data_collection.management.commands import BaseXpressDemocracyClubCsvImporter
 
 
-class Command(BaseXpressDCCsvInconsistentPostcodesImporter):
+class Command(BaseXpressDemocracyClubCsvImporter):
     council_id = "E06000059"
-    addresses_name = "europarl.2019-05-23/Version 1/Democracy_Club__23May2019dorset.tsv"
-    stations_name = "europarl.2019-05-23/Version 1/Democracy_Club__23May2019dorset.tsv"
-    elections = ["europarl.2019-05-23"]
+    addresses_name = "parl.2019-12-12/Version 1/merged.tsv"
+    stations_name = "parl.2019-12-12/Version 1/merged.tsv"
+    elections = ["parl.2019-12-12"]
     csv_delimiter = "\t"
-    station_postcode_search_fields = [
-        "polling_place_postcode",
-        "polling_place_address_4",
-        "polling_place_address_3",
-        "polling_place_address_2",
-    ]
+    allow_station_point_from_postcode = False
```

Comment:
With this one, we've got so many corrections for cases where the map based on postcode wasn't quite right, I'm tempted to enable the maps on the basis we've probably already solved most of the problems that would get reported.. what do you reckon?

Reply:
Yep, I think it's worth a try. There are a lot of polling stations, so still room for more errors, but it will be interesting to see if there's any difference in reaction.